### DISCUSSION TITLE
Notepad++ is automatically re-launched with admin privileges after upgrade

### DISCUSSION THREAD

**Discussion by sryze** [chocolatey-community/chocolatey-packages#2561] [Q&A] [answered]
If Notepad++ is open by a regular user when Chocolatey upgrades the `notepadplusplus` package, it is restarted with Administrator privileges after the package upgrade is finished. I think this is a bug.

**Comment by gep13** [comment, marked as answer]
@sryze thanks for raising this discussion.

This is actually being tracked in this issue:

https://github.com/chocolatey-community/chocolatey-packages/issues/2334

**Reply by gep13** [reply]
As a follow up, a [PR](https://github.com/chocolatey-community/chocolatey-packages/pull/2566) was raised to address the immediate problem.
